### PR TITLE
fixing link

### PR DESCRIPTION
**Pull request by 4c74356b41**
I think it should be like that

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@4c74356b41__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

File: includes/virtual-networks-create-vnet-arm-pportal-include.md [hunk 1]
```diff
@@ -33,4 +33,4 @@ To create a VNet based on the scenario above by using the Azure preview portal,
 
 10. Notice the list of subnets, as shown in the figure below.
 
-	![List of subnets in VNet](./media/virtual-networks-create-vnet-arm-pportal-include/vnet-create-arm-pportal-figure7.png)
```
Comment (4c74356b41) [inline includes/virtual-networks-create-vnet-arm-pportal-include.md]: I didn't change anything here, no idea why it marked that as changed

**Comment by tysonn** [comment]
@4c74356b41  - I made a pull request to your fork with the correct update for this :)

**Comment by 4c74356b41** [comment]
Oh sorry, my bad =\

**Comment by 4c74356b41** [comment]
hello?

**Pull request merged**